### PR TITLE
Add EUID Functionality to Demo Site for CSTG with Prebid.js

### PR DESCRIPTION
**Pull request by ashleysmithTTD**
Adding a new page ( http://unifiedid.com/examples/euid-cstg-prebid-example ) to extend the ability to also generate an EUID advertising token using Prebid.js.  Keeping EUID and UID2 functionality on separate pages. EUID site will move to EUID docs once that site is officially hosted

### CODE REVIEW

File: static/examples/cstg-prebid-example/index.html [hunk 1]
```diff
@@ -9,67 +9,182 @@
...
+                "887": true,    // uid2
+                "95": true,     // lotamePanoramaId
+                "301": true,    // zeotapIdPlus
+                "91": true,     // criteo
+                "737": true,    // amxId
+                "58": true,     // 33acrossId
+              }
+            }
+          }
+        }
```
Comment (ashleysmithTTD) [inline static/examples/cstg-prebid-example/index.html:123]: Design decision here - for UID2, technically all that needs to be sent is "consentData" -> "getTCData" -> "gdprApplies" but if `consentData` does not have some other properties, the code fails.  So to make it clear what is changing between UID2 and EUID with the "consentManagement" prop, I thought it best to keep the entire object defined and only change if "gdprApplies" is true or false based on the idType

File: static/examples/cstg-prebid-example/prebid.js
Comment (ashleysmithTTD) [inline static/examples/cstg-prebid-example/prebid.js:1]: prebid changes are just importing a more updated version of prebid.js

**Review by lionell-pack-ttd — APPROVED** [review]

**Review by lionell-pack-ttd — CHANGES_REQUESTED** [review]
I hit approve but I realised we're still pointing at the UID2 operator. I think for completeness we should point to EUID integ and create new CSTG keys for this site. I'll add details in comments.

File: static/examples/euid-cstg-prebid-example/index.html [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+    <meta charset="UTF-8" />
+    <title>EUID Prebid.js Client-Side Integration Example</title>
+    <link rel="stylesheet" type="text/css" href="./app.css" />
+    <link rel="shortcut icon" href="/img/favicon.ico" />
+    <script async src="./prebid.js"></script>
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+    <script>
+      console.log('Initializing example.')
+      window.euid_example_settings = {
+        EUID_BASE_URL: 'https://operator-integ.uidapi.com',
```
Comment (lionell-pack-ttd) [inline static/examples/euid-cstg-prebid-example/index.html:13]: This is the UID2 integ operator - I think for thoroughness here we should point to EUID infra. The EUID operator is at https://integ.euid.eu

File: static/examples/euid-cstg-prebid-example/index.html [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+        SERVER_PUBLIC_KEY: 'UID2-X-I-MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEo+jcPlk8GWn3iG0R5Il2cbFQI9hR3TvHxaBUKHl5Vh+ugr+9uLMiXihka8To07ETFGghEifY96Hrpe5RnYko7Q==',
```
Comment (lionell-pack-ttd) [inline static/examples/euid-cstg-prebid-example/index.html:14]: You'll need to generate new keys for EUID. I'll send you details.

**Review by lionell-pack-ttd — APPROVED** [review]

**Pull request merged**